sdk/cognitiveservices/Vision.ComputerVision/CHANGELOG.md

```diff
@@ -1,11 +1,25 @@
 # Release History
 
-## 6.1.0-beta.1 (Unreleased)
+## 7.0.0 (2021-04-12)
+
+### Added
+
+* Added model versioning support
```

Comment:
Should we be explicit about which API version this will be serving?
We has this question many times from customers.

https://github.com/Azure/azure-sdk-for-net/blob/Microsoft.Azure.CognitiveServices.Vision.ComputerVision_6.0.0/sdk/cognitiveservices/Vision.ComputerVision/CHANGELOG.md

Reply:
Yes, it's fair to add "Adding support for 3.2 in the ChangeLog"

Reply:
Agreed. Added corresponding REST API version information to the changelog.